wander-connect-sdk/src/components/iframe/iframe.component.ts

```diff
@@ -213,6 +213,7 @@ export class Iframe {
 
     const iframe = document.createElement("iframe");
     iframe.className = "iframe";
+    iframe.allow = "camera *;";
```

Comment:
We add camera permissions added to the iframe so new SDK release needs to be made for this.